src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class ColorSupplier {
+
```

Comment:
please look through your classes and remove empty lines after class declarations

Reply:
removed